src/components/NavItems.js

```diff
@@ -33,6 +40,8 @@ export function NavItems() {
           </Button>
         )
       )}
+      {/* uncomment this line to test <Logout/>. Button is temporary */}
+      {/* {isLoggedIn && <Logout />} */}
```

Comment:
Uncomment this code to test the `<Logout/>` component. Button is temporary.

Reply:
Looks Good, but here rather than uncomment that, we can show logout when user is logged. the code may look like this one; `
	{navItems.map((item) => {
        if(item.navLinks === 'Login' && isLoggedIn){
          console.log('yes logged in')
          return <Logout/>
        }
				return item.menuLinks.length > 0 ? (
					<DropDown
						key={item.navLinks}
						navLink={item.navLinks}
						menuLinks={item.menuLinks}
					/>
				) : (
					<Button key={item.navLinks} color="inherit">
						<Link
							to={`/${item.navLinks}`}
							style={{ textDecoration: 'none', color: 'inherit' }}
						>
							{item.navLinks}
						</Link>
					</Button>
				);
			})}
`

Reply:
I made the temporary button but @IreneS1 is going to make an avatar drop down with a favorites and logout section in a latter pull request @